canonical-websites/www.ubuntu.com
# 2826 data privacy page

## Done

- Add new data privacy page
- Archive old privacy policy page
- Add redirects from old privacy policy page to new data privacy page
- Update cookie link in footer

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal/dataprivacy>
- Make sure page matches [the copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit#)
- Make sure <http://0.0.0.0:8001/legal/terms-and-policies/privacy-policy> redirects to <http://0.0.0.0:8001/legal/dataprivacy>
- Make sure footer contains a link to the data privacy page

## Issue / Card

Fixes #2826
